python/masonite/Kernel.py

```diff
@@ -0,0 +1,109 @@
+from masonite.foundation import response_handler
+from masonite.storage import StorageCapsule
+from masonite.auth import Sign
+from masonite.environment import LoadEnvironment
+from masonite.utils.structures import load
+from masonite.utils.location import base_path
+from masonite.middleware import (
+    SessionMiddleware,
+    EncryptCookies,
+    LoadUserMiddleware,
+    MaintenanceModeMiddleware,
+    VerifyCsrfToken,
```

Comment:
initially the version inside this implementation was used, why not using the one provided by `masonite`

Reply:
It is basically. This class is in new projects and inherits the Masonite class basically just for the purpose of being able to exclude routes from csrf